regression/esbmc/github_330_4/test.desc

```diff
@@ -0,0 +1,4 @@
+CORE
+main.c
+
```

Comment:
Did you forget the `--memory-leak-check` here?

Reply:
Fixed.